### DISCUSSION TITLE
Things before I start

### DISCUSSION THREAD

**Discussion by shashankx86** [fltk-rs/fltk-rs#1540] [Q&A]
I am making a GUI clipboard manger for linux
its pop up when Super + V is pressed

so, is possible to implement 

1.daemon like thing (keeps running is background), when super + v is pressed the window pops up
2.setting widow so that its always stays on top

thnx

**Comment by MoAlyousef** [comment]
1- This is doable. You would have to create a daemon app which would listen to the shortcut, and when pressed it would spawn an FLTK window with the settings etc. 

2- FLTK doesn’t expose such functionality. You would have to use the OS api to set your window to be always on top. fltk-rs has a `Window::set_on_top()` available for macos since it was easy to implement. Wayland doesn’t support it AFAIK, on X11 it depends on the _NET_WM_STATE_ABOVE hint so not always available. Windows shoud be easy as well.

**Reply by shashankx86** [reply]
I am making it for wayland
